sycl/include/CL/sycl/intel/fpga_device_selector.hpp

```diff
@@ -0,0 +1,51 @@
+//==- fpga_device_selector.hpp --- SYCL FPGA device selector shortcut  -----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl.hpp>
+
+namespace cl {
+namespace sycl {
+namespace intel {
+
+class platform_selector : public default_selector {
```

Comment:
Should this derive from device_selector instead of default_selector?

Reply:
good point. I will fix this in next patch.